Toml11 conversion (#994)
GMLC-TDC/HELICS: CONTRIBUTORS.md

### [CLI11](https://github.com/CLIUtils/CLI11)
CLI11 is a command line parser and was used as a replacement for boost::program_options. Some modifications used in HELICS were committed to the CLI11 library and others are in the process of doing so. The modified single header library is included in the HELICS source code. The project was created by Henry Schreiner. A modified version of the CLI11 header is included with a HELICS install,  Changes are currently being worked into upstream repositories.  CLI11 is available under a [3-Clause BSD](https://github.com/CLIUtils/CLI11/blob/master/LICENSE) license.

### [toml11](https://github.com/ToruNiina/toml11)

>   toml11 is a C\++11 header-only toml parser/encoder depending only on C++ standard library.  Compatible to the latest version of TOML v0.5.0. toml11 is licensed under [MIT](https://github.com/ToruNiina/toml11/blob/master/LICENSE) license. HELICS includes it through a submodule of a library [fork](https://github.com/GMLC-TDC/toml11) until the changes can be worked upstream.  

Reviewer: ```suggestion
  toml11 is a C\++11 header-only toml parser/encoder depending only on the C++ standard library.  Compatible with the latest version of TOML v0.5.0. toml11 is licensed under the [MIT](https://github.com/ToruNiina/toml11/blob/master/LICENSE) license. HELICS includes it through a submodule of a library [fork](https://github.com/GMLC-TDC/toml11) until the changes can be merged upstream.  
```
